pkg/nimble/Makefile.include

```diff
@@ -93,15 +93,27 @@ ifneq (,$(filter nimble_netif,$(USEMODULE)))
 
   # configure NimBLE's internals
   NIMBLE_MAX_CONN ?= 3
-  CFLAGS += -DMYNEWT_VAL_MSYS_1_BLOCK_SIZE=264
   CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MAX_NUM=$(NIMBLE_MAX_CONN)
   CFLAGS += -DMYNEWT_VAL_BLE_MAX_CONNECTIONS=$(NIMBLE_MAX_CONN)
-  # NimBLEs internal buffer need to hold one IPv6 MTU per connection
-  # for the internal MTU of 256 byte, we need 10 mbufs per connection...
+
+  # the maximum fragment size that we can receive. For maximum efficiency this
+  # should be equal to the maximum configured link layer packet size.
+  # WARNING: this value MUST never be larger than MYNEWT_VAL_BLE_LL_MAX_PKT_SIZE
+  CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MPS=251
```

Comment:
Can we add an error message somewhere in code? If it shouldn't be larget then it should fail at compilation I would say.

Reply:
Difficult situation: 
1. this condition does only apply, if we use the NimBLE controller. If we would build NimBLE against another controller, it would not apply anymore...
2. structural, this check would need to go into NimBLE and not RIOT. But I have no idea currently how to implement it there in a clean way.
3. As we define this value here as a constant and as we don't allow to override this from the build system, I think we should be safe without that check. If anyone starts to mess with this value, I'd say its up to them to read the warning above.

Reply:
Ok, I agree this belongs in nimble.